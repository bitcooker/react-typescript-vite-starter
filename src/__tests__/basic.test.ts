import { describe, expect, it } from 'vitest'

describe('basic', () => {
  it('multiplication', () => {
    expect(5 * 5).toMatchSnapshot()

    expect(5 * 5).toStrictEqual(25)
  })
})
