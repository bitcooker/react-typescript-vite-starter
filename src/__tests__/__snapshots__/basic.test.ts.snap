// Vitest Snapshot v1

exports[`basic > multiplication 1`] = `25`;
